analysis/tc_tackle_testapp_public_binary.go

```diff
@@ -21,22 +21,8 @@ var TackleTestappPublicBinary = TC{
 	},
 	Binary: true,
 	Analysis: api.Analysis{
-		Effort: 15,
+		Effort: 1,
 		Issues: []api.Issue{
-			{
-				Category:    "mandatory",
-				Description: "Java native libraries (JNI, JNA)",
-				Effort:      7,
-				RuleSet:     "cloud-readiness",
-				Rule:        "jni-native-code-00000",
-				Incidents: []api.Incident{
-					{
-						File:    "/shared/bin/maven/java-project/src/main/java/io/konveyor/demo/config/ApplicationConfiguration.java",
-						Line:    17,
-						Message: "Java native libraries might not run in a cloud or container environment.",
```

Comment:
This seems to be okay - looks like a false positive from [this rule](https://github.com/konveyor/rulesets/blob/main/default/generated/cloud-readiness/69-jni-native-code.windup.yaml#L26) that was happening [here](https://github.com/konveyor/tackle-testapp-public/blob/main/configuration-utils/src/main/java/io/konveyor/demo/config/ApplicationConfiguration.java#L17), so it not appearing anymore seems sane to me.